checklist_plus/utils.py

```
@@ -0,0 +1,122 @@
+        # Auto brands that look like nouns
+        'ford', 'dodge', 'ram', 'jeep', 'saturn', 'mercury', 'genesis', 'infinity', 'acura',
+        'smart', 'mini', 'fiat', 'seat', 'skoda', 'dacia', 'alpine', 'lotus', 'mclaren',
+
+        # Food/Beverage brands that look like nouns
+        'subway', 'dominos', 'papa', 'johns', 'dairy', 'queen', 'burger', 'king', 'white',
+        'castle', 'five', 'guys', 'shake', 'shack', 'chipotle', 'panera', 'bread', 'olive',
+        'garden', 'red', 'lobster', 'outback', 'steakhouse', 'applebees', 'fridays', 'ruby',
+        'tuesday', 'buffalo', 'wild', 'wings', 'hooters', 'dennys', 'ihop', 'waffle', 'house',
+
+        # Retail brands that look like nouns
+        'target', 'gap', 'old', 'navy', 'banana', 'republic', 'american', 'eagle', 'express',
```

Ruff (0.13.1)

Sets should not contain duplicate item `'target'`

Remove duplicate item

(B033)